Blog post - Alicja Suska - 'What happens when a successful product gets its first design team...' (#1558)
sourcegraph/about: blogposts/2020/what-happens-when-a-successful-product-gets-its-first-design-team.md

---
title: "What happens when a successful product gets its first design team..."
author: Alicja Suska
authorUrl: https://alicjasuska.com

> publishDate: 2020-09-11T18:00-07:00

Reviewer: change publishDate to today's date

Author: Changed

tags: [blog]
slug: what-happens-when-a-successful-product-gets-its-first-design-team
published: true
---

We are happy to announce that the Sourcegraph design team has grown from 0 to 3! 🎉

> The early stages of a new team are full of changes and unexpected challenges. I would like to share with you how our first months of working together look like, what we have accomplished, and what are our plans for the future.

Reviewer 2: ```suggestion
The early stages of a new team are full of changes and unexpected challenges. I would like to show you what our first months of working together looked like, what we have accomplished, and our plans for the future.
```

## First 90 days

> From the very beginning, we knew that to set our team up for success, we need to carefully plan the next 3 months. There was a lot to focus on: processes, design debt, new initiatives, etc. We had to choose which areas to take care of first and where we can have the biggest impact.

Reviewer 2: ```suggestion
From the very beginning, we knew that we needed to carefully plan the next 3 months to set our team up for success. There was a lot to focus on: processes, design debt, new initiatives, etc. We had to choose which areas to take care of first and where we could have the biggest impact.
```
